docs/mllib-clustering.md

```diff
@@ -507,6 +507,10 @@ must also be $> 1.0$. Providing `Vector(-1)` results in default behavior
 $> 1.0$. Providing `-1` results in defaulting to a value of $0.1 + 1$.
 * `maxIterations`: The maximum number of EM iterations.
 
+*Hint*: It is important to do enough iterations.  In early iterations, EM often has useless topics,
```

Comment:
nit: I prefer "Note" instead of "Hint" since I haven't seen "Hint" anywhere else in user guides, but feel free to leave if you like better